RecoLocalCalo/HGCalRecAlgos/interface/RecHitTools.h

```diff
@@ -0,0 +1,36 @@
+#ifndef __RecoLocalCalo_HGCalRecAlgos_RecHitTools_h__
+#define __RecoLocalCalo_HGCalRecAlgos_RecHitTools_h__
+
+#include <array>
+#include <cmath>
+
+class HGCalGeometry;
+class HGCalDDDConstants;
+class DetId;
+
+namespace edm {
+  class Event;
+  class EventSetup;
+}
+
+namespace hgcal {
+  class RecHitTools {    
+  public:
+  RecHitTools() : geom_({ {nullptr,nullptr} }), ddd_({ {nullptr,nullptr} }) {}
+    ~RecHitTools() {}
+
+    void getEvent(const edm::Event&);
+    void getEventSetup(const edm::EventSetup&);
+    
+    std::float_t getSiThickness(const DetId&) const;
```

Comment:
@lgray - as Slava questions - we should avoid std::float_t at this point. Could you change these to floats? Thanks!